include/llvm/Config/config.h

```diff
@@ -18,7 +18,7 @@
 #define ENABLE_CRASH_OVERRIDES 1
 
 /* Define to 1 if you have the `backtrace' function. */
-#if __has_include(<execinfo.h>)
+#if __has_include(<execinfo.h>) && !defined(__ANDROID__)
```

Comment:
> Also, do we have a #define somewhere for the minimum Android version we support?

No, I don't think so.

> Could we do a check against that, so that this starts working when we increase the minimum supported version?

[Android has `__ANDROID_API__` for the API being built against instead](https://android.googlesource.com/platform/bionic/+/refs/heads/main/libc/include/stdio.h#61), which our builds should define, so I could make this `#if __has_include(<execinfo.h>) || __ANDROID_API__ >= 33` if you prefer.

> Could you stick a comment next to this explaining that although Android has it, we need to support older Android APIs.

Will do, if we don't check the Android API.

Reply:
Presumably it'd need to be
```
#if __has_include(<execinfo.h>) && (!defined(__ANDROID__) || __ANDROID_API__ >= 33)
```
but yes, if it's got the API level check then we don't need the comment.